pkg/cmd/target/set.go

```diff
@@ -35,12 +44,31 @@ var TargetSetCmd = &cobra.Command{
 			log.Fatal(err)
 		}
 
-		pluginList, err := apiclient_util.GetProviderList()
+		serverConfig, res, err := apiClient.ServerAPI.GetConfigExecute(apiclient.ApiGetConfigRequest{})
+		if err != nil {
+			log.Fatal(apiclient_util.HandleErrorResponse(res, err))
+		}
+
+		providersManifest, err := manager.NewProviderManager(manager.ProviderManagerConfig{
+			RegistryUrl: serverConfig.RegistryUrl,
+		}).GetProvidersManifest()
 		if err != nil {
 			log.Fatal(err)
 		}
 
-		selectedProvider, err := provider.GetProviderFromPrompt(pluginList, "Choose a provider", false)
+		if providersManifest == nil {
```

Comment:
I suggest that we don't fatal this but just log the error because the registry might be offline but that shouldn't prevent you from setting a target.

Reply:
![Screenshot 2024-09-03 at 14 58 01](https://github.com/user-attachments/assets/7ba94ff0-2133-4891-83af-984a9af56c0e)

I suggest the message be `Could not get target manfiest. Can't check for new providers to install.`

Reply:
P.S. I changed the registry url to something invalid to test this